assets/js/components/Work/Tabs/Preservation/FileSetForm.jsx

```diff
@@ -3,56 +3,61 @@ import PropTypes from "prop-types";
 import UIFormInput from "@js/components/UI/Form/Input.jsx";
 import UIFormField from "@js/components/UI/Form/Field.jsx";
 import UIFormSelect from "@js/components/UI/Form/Select.jsx";
-import { FILE_SET_ROLES } from "@js/services/global-vars";
-import Error from "@js/components/UI/Error";
+import { useCodeLists } from "@js/context/code-list-context";
+import { CodeListProvider } from "@js/context/code-list-context";
 
 function WorkTabsPreservationFileSetForm({ s3UploadLocation }) {
+  const codeLists = useCodeLists();
+
   return (
     <>
       {s3UploadLocation && (
         <div>
-          <UIFormField label="Accession number">
-            <UIFormInput
-              isReactHookForm
-              required
-              label="Accession number"
-              data-testid="fileset-accession-number-input"
-              name="accessionNumber"
-              placeholder="accession number"
-            />
-          </UIFormField>
+          <CodeListProvider>
```

Comment:
Not sure we need the `CodeListProvider`?   Seems to work without it but probably not a big deal if it's not throwing errors.